.github/workflows/api-dotnetcore.yml

```diff
@@ -85,3 +96,20 @@ jobs:
           env_vars: C#
           # Specify whether or not CI build should fail if Codecov runs into an error during upload
           fail_ci_if_error: true
+
+      - name: SonarScanner for .NET 5 with pull request decoration support
+        uses: highbyte/sonarscan-dotnet@2.0
+        if: ${{ github.event_name == 'push' }}
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          dotnetBuildArguments: ${{env.working-directory}}
+          dotnetTestArguments: ${{env.working-directory}} --logger trx --collect:"XPlat Code Coverage" -- DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=opencover
+          # Optional extra command arguments the the SonarScanner 'begin' command
+          sonarBeginArguments: /d:sonar.cs.opencover.reportsPaths="**/TestResults/**/coverage.opencover.xml" -d:sonar.cs.vstest.reportsPaths="**/TestResults/*.trx"
+          # The key of the SonarQube project
+          sonarProjectKey: 483bc59d-f9ef-428c-866e-581d9ea69eab
+          # The name of the SonarQube project
+          sonarProjectName: PIMS-API
```

Comment:
I suppose this means that we are moving to just one sonar project for the api, instead of the one-per-env setup that we currently have (ie pims-api[dev])

Reply:
I thought about this.  It doesn't make sense to me to have multiple projects for different branches.  It's one project.  We don't want to have to apply rules multiple times to deal with the different branches.  Any inconsistencies between branches should show up in the report history.
The only annoying outcome possibly would be if someone started pushing changes to other branches without adding them to the `dev` branch.

Reply:
The problem with it from my perspective is timing I guess, if we are about to push to PROD, and Stanley goes to check if we have any outstanding vulnerabilities, and he sees that we have a whole bunch from an unstable build just pushed to DEV he is unlikely to want to release to PROD, even though the code being released to PROD may not have those issues.

Reply:
Same is possible in reverse, we have a whole bunch of vulnerabilities in PROD, someone fixes them in DEV, Stanley looks at the report and everything looks good, and so he releases code with known vulnerabilities to PROD.

Reply:
I think most of those kind of issues can be resolved through review of the SonarQube Activities.  Each report should be reviewable.

Reply:
Additionally at a future date we could find a way to make a failure on merge show up in GitHub.

Reply:
We can also review and split these up at a later date if required.  It would only require a modification to these actions and the creation of the projects in SonarQube.